WordPress/src/main/java/org/wordpress/android/ui/prefs/categories/CategoriesViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.ui.prefs.categories
+
+import androidx.lifecycle.ViewModel
+import javax.inject.Inject
+
+class CategoriesViewModel @Inject constructor() : ViewModel() {
```

Comment:
Suggestion (💡): Since the corresponding activity and fragment naming contains the word `List` within, like `CategoriesListActivity` and `CategoriesListFragment`, I suggest that the view model is renamed to contain this word as well. As such it will become `CategoriesListViewModel` instead. Wdyt?

PS: Same applies for its accompanying test.

Reply:
Good catch, I was back-n-forth on the name. Guess I forget to refactor this one back.